### ISSUE TITLE
The m/z values ​​are displayed differently. (expressed as a negative number)

### ISSUE THREAD

**Issue by jiwony9** [mzmine/mzmine#1985]
### Basic information

* My operating system and version:
* My mzmine version: mzmine 4.1.0

### What happened

![image](https://github.com/user-attachments/assets/81e02202-3761-4951-bfd8-20726c53df90)

I was processing using mzmine (water.raw data -> convert using waters data connection & MS convert)
but the m/z values are displayed differently.

ex) original value is 436.04, but it is displayed -434.04.

batch and raw files : https://drive.google.com/file/d/1q3AllsKkCyLRukXDcfjw6UutquoG4aH1/view?usp=sharing

* Please describe what happened and how it differs from what you expected to happen.
* Please include all parameter values for the mzmine modules you used. 
* Please upload screenshots of any errors **and** your log/mzmine.log file.
* **Please upload a standalone mzmine project file (or a batch + raw files) that demonstrates the problem you are reporting.** You can use Google Drive, Dropbox, or a similar repository to upload the project.
* Issues opened without an example project are unlikely to be processed

**Comment by SteffenHeu** [comment]
Please upload a standalone mzmine project file (or a batch + raw files) that demonstrates the problem you are reporting. You can use Google Drive, Dropbox, or a similar repository to upload the project.
Issues opened without an example project are unlikely to be processed

**Comment by robinschmid** [comment]
Are the masses also displayed negative in mzmine? 
This could be a conversion issue.

**Comment by jiwony9** [comment]
Both files (project file and raw file) have been uploaded. Thank you for checking. This same problem also occurs when using Prgenic QI.

https://groups.google.com/g/molecular_networking_bug_reports/c/JyOvkM7wjg4/m/M8rkQX70BAAJ?utm_medium=email&utm_source=footer

**Comment by jiwony9** [comment]
> Are the masses also displayed negative in mzmine? This could be a conversion issue.

No, they do not show up as negative in mzmine and exported csv files. It only appears as a negative in gnps.

**Comment by SteffenHeu** [comment]
Hi jowony, 
i opened the project in mzmine and i can see indication of negative masses. Therefore I would say it is a gnps issue, so better contact them.
